File: boj/brute_force/13225/13225.rs
use std::io;

fn main() {
    let mut line = String::new();
    io::stdin().read_line(&mut line).expect("wrong io");
    let mut n = line.trim().parse::<i32>().unwrap();
    loop {
        let mut line = String::new();
        io::stdin().read_line(&mut line).expect("wrong io");
        let num = line.trim().parse::<i32>().unwrap();
        let mut divisors = vec![];
        for i in 1..num + 1 {
            if num % i == 0 {
                divisors.push(i);
            }
        }
        println!("{} {}", num, divisors.len());
        n -= 1;
        if n == 0 {
            break;
        }
    }
}

File: boj/brute_force/13225/13225.cpp
#include <bits/stdc++.h>
using namespace std;
int n, num;

int main(){
    cin >> n;
    while(n--){
        cin >> num;
        vector <int> divisors;

        for(int i = 1; i <= num; i++) {
            if(num % i == 0) divisors.push_back(i);
        }
        cout << num << ' ' << divisors.size() << '\n';
    }
}